keyboards/bastardkb/charybdis/3x5/keymaps/default/rules.mk

```diff
@@ -0,0 +1,2 @@
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
```

Comment:
This should go into the keyboard's rules.mk file.  Along with `RGB_MATRIX_SUPPORTED = yes`

Reply:
Done.